src/ImuSensor.cc

```diff
@@ -255,6 +256,41 @@ bool ImuSensor::Update(const std::chrono::steady_clock::duration &_now)
   frame->set_key("frame_id");
   frame->add_value(this->FrameId());
 
+  // Populate covariance
+  for (int i = 0; i < 9; ++i) {
```

Comment:
number 9 as a const ?

Reply:
I don't see a problem, but wouldn't that be less easy to read ?

```c++
const int matrix_3x3_size = 9;
for (int i = 0; i < matrix_3x3_size; ++i)
...
```